desktop-exporter/trace_data.go

```diff
@@ -0,0 +1,40 @@
+package desktopexporter
+
+import (
+	"time"
+)
+
+func (trace *TraceData) GetTraceSummary() (*TraceSummary, error) {
+	duration, err := trace.getTraceDuration()
+	if err != nil {
+		return nil, err
+	}
+
+	return &TraceSummary{
+		TraceID:    trace.Spans[0].TraceID,
+		SpanCount:  uint32(len(trace.Spans)),
+		DurationMS: duration.Milliseconds(),
+	}, nil
+
+}
+
+func (trace *TraceData) getTraceDuration() (time.Duration, error) {
+	if len(trace.Spans) < 1 {
+		return 0, ErrEmptySpansSlice
+	}
```

Comment:
Consider simplifying this by returning a duration of `0` if there are no spans to eliminate the error condition from these methods